aiohttp/web_reqrep.py

```diff
@@ -774,7 +774,7 @@ def __init__(self, *, body=None, status=200,
         self.set_tcp_cork(True)
 
         if body is not None and text is not None:
-            raise ValueError("body and text are not allowed together.")
+            raise ValueError('body and text are not allowed together')
```

Comment:
I thought the difference in quotes usage was due to different contributors. I'll change it back